app/src/main/res/layout-sw600dp-port/ongoing_story_card.xml

```diff
@@ -84,7 +82,6 @@
           android:text="@{viewModel.ongoingStory.topicName}"
           android:textAllCaps="true"
           android:textColor="@color/oppiaPrimaryText"
```

Comment:
`android:textColor="@color/oppiaPrimaryText"`
You can remove this line too because this is already mentioned in styles.xml

Check this:
`Body -> TextView.Common1`